open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -839,12 +839,12 @@ def __create_hackathon(self, creator, context):
 
         return new_hack
 
-    def __get_pre_allocate_interval(self, hackathon):
+    def __get_pre_allocate_interval(self, hackathon, index):
         interval = self.get_basic_property(hackathon, HACKATHON_CONFIG.PRE_ALLOCATE_INTERVAL_SECONDS)
         if interval:
             return int(interval)
         else:
-            return 300 + hackathon.id * 10
+            return 300 + index * 20
```

Comment:
The purpose of old code to add `hackathon.id * 10` is to make intervals different for hackathons to avoid those jobs executed at the same time. So you don't need to pass `index`. Use `random` if you want to keep the same purpose.